app/directives/schedule/room-row.js

```diff
@@ -16,6 +16,17 @@ define(['app','lodash', 'text!./room-row.html','text!../forms/edit/room-dialog.h
         },
         link: function($scope,$element) {
 
+          $scope.room.reqCode = getCode($scope.room);
+            function getCode(room){
+              let code ='';
+
+              if(room.hybrid) code += 'H';
+              if(room.interpretationRemote) code += 'R';
+              if(room.interpretationOnSite) code += 'O';
+              if(room.interpretationBooths) code += `${room.interpretationBooths}`;
+
+              return code? `${code}`:'';
+          }
```

Comment:
I think it woud have been better to make them part of the HTML instead with a tooltip to explain what each letter mean. but not big deal

 eg:
```html
<span ng-if="room.hybrid" tooltip="Hybrid capabilities">H</span>
<span ng-if="room.interpretationRemote" tooltip="Spport Remote Interpetation">R</span>
<span ng-if="room.interpretationOnSite"  tooltip="Spport Onsite Interpetation">O</span>
<span ng-if="room.interpretationBooths" tooltip="{{room.interpretationBooths}} Interpetation booths avilable">{{room.interpretationBooths}}</span>
```